src/main/java/DataManipulation.java

```diff
@@ -0,0 +1,89 @@
+  import static java.util.Arrays.asList;
+  import java.io.BufferedReader;
+  import java.io.FileNotFoundException;
+  import java.io.FileReader;
+  import java.io.FileWriter;
+  import java.io.IOException;
+  import java.util.List;
+  import org.apache.arrow.memory.RootAllocator;
+  import org.apache.arrow.vector.FieldVector;
+  import org.apache.arrow.vector.VarCharVector;
+  import org.apache.arrow.vector.VectorSchemaRoot;
+
+public class DataManipulation {
+  public static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+  VarCharVector nameVector = null;
+  VarCharVector teamVector = null;
+  VarCharVector positionVector = null;
+  VarCharVector heightVector = null;
+  VarCharVector weightVector = null;
+  VarCharVector  ageVector = null;
```

Comment:
You don't need these assignement here. You can make everything in the readCSVFile method since you are not going to use it outside the method scope